kyuubi-common/src/main/scala/org/apache/kyuubi/operation/FetchIterator.scala

```diff
@@ -109,3 +109,26 @@ class IterableFetchIterator[A](iterable: Iterable[A]) extends FetchIterator[A] {
     }
   }
 }
+
+object FetchIterator {
+
+  def from[A](iter: Iterator[A]): FetchIterator[A] = new FetchIterator[A] {
```

Comment:
is this so common to have this name?

Reply:
OK, I modified it to `fromIterator`, just like the `Future.fromTry` API

Reply:
shouldn't it be as same as IterableFetchIterator?